linode/manager
# Change: Observe User Preferences for Linodes Landing View

## Description

Uses User Preferences for Linodes View

## Type of Change
- Breaking change ('break', 'deprecate')

## Applicable E2E Tests

N/A

## Note to Reviewers

_!!DEPRECATED FUNCTIONALITY!!_

Before the priority for showing grid or list view was this

1. URL Query Param (show whatever is in the param)
2. Viewport width (show grid view on xs breakpoints)
3. Local storage (use whatever is in LS)
4. Linode Count (show grid view if user has < 3 Linodes, otherwise show list view)

But since we now have implemented responsive tables, it was more effort than necessary to still honor this hierarchy, so instead this PR implements the following logic.

1. URL Query Param (show whatever is in the param)
2. User Preference (show whatever is in user preference, and default to list view)

## To Test

Toggle all existing user preferences (theme, spacing, linode view, and linode group by tag). Make sure they're not overwriting each other and PUT to the API with the correct payload.
